### PR TITLE
Add hashing function

### PR DESCRIPTION
**Pull request by bryantfhayes**
hashes the global dictionary to prevent Nick's "hit" count from winning!

### CODE REVIEW

File: cogvents.py [hunk 1]
```diff
@@ -111,9 +127,11 @@ def main():
     # ONLY ADD/MOVE LISTERNS UNDER HERE!   
     AddListener('Event_Start', fnShitGetsCrayFoSheezey)
     AddListener('Event_Start', fnDynamite)
+    AddListener('Event_Start', HashThisShit)
```
Comment (Lapitz) [inline cogvents.py]: I reject your blatant attempt at winning. Hashing the names will fill them with long strings containing countless 'a' and 'e' vowels. Which you picked to run right before the vowelLimiter function.

Comment (bryantfhayes) [inline cogvents.py]: What are you talking about? All it does is make a short encryption string

Comment (Lapitz) [inline cogvents.py]: By short you mean 512 bit long hex strings for every dictionary entry?

**Comment by bryantfhayes** [comment]
bummer

**Pull request closed without merging**